TramsDataApi/DatabaseModels/A2BSchoolLease.cs

```diff
@@ -7,14 +7,15 @@ namespace TramsDataApi.DatabaseModels
     public class A2BSchoolLease
     {
         [Key]
-        public string SchoolLeaseId {get; set;}
-        
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
this should be a int, I'm not sure how ef core/ ms sql would create an identity string column, but I would imagine that it could cause issues...